<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Herramienta de Transmision de Video</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="select_codec">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>h264</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>h265</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VP8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VP9</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="ip_dst">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.1.2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP de destino:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="port_dst">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>5004</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_cargar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(114, 159, 207);</string>
    </property>
    <property name="text">
     <string>Cargar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Codec:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccionar video [*.yuv]:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Tx">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(114, 159, 207);</string>
    </property>
    <property name="text">
     <string>Transmitir Video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Rx">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(114, 159, 207);</string>
    </property>
    <property name="text">
     <string>Recibir PCAPs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_stop">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(239, 41, 41);</string>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QLabel" name="tx_label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Esperando ...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_metrics">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>200</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(138, 226, 52);</string>
    </property>
    <property name="text">
     <string>Calcular Metricas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Throughput promedio [kbps]:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>550</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Delay promedio [ms]:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>580</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Packet Loss [%]:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="pl_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>576</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="thr_label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="delay_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>540</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_plots">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>550</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(138, 226, 52);</string>
    </property>
    <property name="text">
     <string>Generar Gr√°ficas </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Piso No.</string>
    </property>
   </widget>
   <widget class="QListWidget" name="select_piso">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>41</width>
      <height>71</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="time_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>620</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>620</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiempo de Tx:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="photo1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>cuenca.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Puerto:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_sync">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>600</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(239, 41, 41);</string>
    </property>
    <property name="text">
     <string>Sincronizar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inputFileButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="inputFileLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QListWidget" name="select_tool">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>GStreamer</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FFMPEG</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccionar Herramienta de Tx:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="play_video">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>600</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(239, 41, 41);</string>
    </property>
    <property name="text">
     <string>Play Video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_plots_tot">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>550</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(138, 226, 52);</string>
    </property>
    <property name="text">
     <string>GRAFICAS</string>
    </property>
   </widget>
   <widget class="QListWidget" name="select_sim">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>41</width>
      <height>91</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Exp No.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_encode">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(196, 160, 0);</string>
    </property>
    <property name="text">
     <string>Codificar Video</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ip_src">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.1.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>IP de origen:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>240</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Uso de CPU [%]:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="cpu_label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>245</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <zorder>photo1</zorder>
   <zorder>select_codec</zorder>
   <zorder>ip_dst</zorder>
   <zorder>label</zorder>
   <zorder>port_dst</zorder>
   <zorder>Btn_cargar</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>Btn_Tx</zorder>
   <zorder>Btn_stop</zorder>
   <zorder>tx_label</zorder>
   <zorder>Btn_metrics</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>pl_label</zorder>
   <zorder>thr_label</zorder>
   <zorder>delay_label</zorder>
   <zorder>Btn_plots</zorder>
   <zorder>label_8</zorder>
   <zorder>select_piso</zorder>
   <zorder>time_label</zorder>
   <zorder>label_10</zorder>
   <zorder>label_2</zorder>
   <zorder>Btn_Rx</zorder>
   <zorder>Btn_sync</zorder>
   <zorder>inputFileButton</zorder>
   <zorder>inputFileLineEdit</zorder>
   <zorder>select_tool</zorder>
   <zorder>label_11</zorder>
   <zorder>play_video</zorder>
   <zorder>Btn_plots_tot</zorder>
   <zorder>select_sim</zorder>
   <zorder>label_12</zorder>
   <zorder>Btn_encode</zorder>
   <zorder>ip_src</zorder>
   <zorder>label_13</zorder>
   <zorder>label_25</zorder>
   <zorder>cpu_label</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
